pepper-apis/src/main/resources/db-changes/schema/DDP-4911-test-result-event.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200908-test-result-event-type-table">
+        <createTable tableName="test_result_event_type">
+            <column name="test_result_event_type_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true" primaryKeyName="test_result_event_type_pk"/>
+            </column>
+            <column name="test_result_event_type_code" type="varchar(20)">
+                <constraints nullable="false" unique="true" uniqueConstraintName="test_result_event_type_uk"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet author="yufeng" id="20200908-test-result-event-types">
+        <insert tableName="test_result_event_type">
+            <column name="test_result_event_type_code" value="NEGATIVE"/>
+        </insert>
+        <insert tableName="test_result_event_type">
+            <column name="test_result_event_type_code" value="POSITIVE"/>
+        </insert>
+        <insert tableName="test_result_event_type">
+            <column name="test_result_event_type_code" value="OTHER"/>
+        </insert>
+        <insert tableName="test_result_event_type">
+            <column name="test_result_event_type_code" value="ANY"/>
+        </insert>
+    </changeSet>
```

Comment:
Not sure. Some studies might be interested in that. For TestBoston it's sufficient to know that a test result is available because our implementation for TestBoston is to use one single activity.

Reply:
Actually, I was focusing more on the idea of specifying that at test result was POSITIVE or NEGATIVE as an even type. This seems more like content of the test result.

Reply:
I felt we had to key it off of the "content of the test result" like this because of the nature of the problem. We had 3 different report letters for each of the positive/negative/other outcome. So if we were to create different activities for positive/negative/other then I don't see a way to do it unless you actually look at the content of the test result like this. However, we were able to achieve having just one single activity, so I can remove this now.

Reply:
Removed!